### PR TITLE
[Shevtsov Volodymyr] Homework. Lesson 1. Resume.

### PR DESCRIPTION
**Pull request by ShevtsovVolodymyr**

### CODE REVIEW

File: home/Volodymyr Shevtsov/Lesson 1/index_en.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <head>
+        <meta charset="utf-8">
+        <title>Resume</title>
+        <link rel="stylesheet" type="text/css" href="resume.css">
+    </head>
+    <body>
+        <section>
+        
+            <div class="description">
+                <img class="foto" src="myfoto.jpg" width="230" height="230" alt="">
```
Comment (Polik32) [inline home/Volodymyr Shevtsov/Lesson 1/index_en.html]: photo

File: home/Volodymyr Shevtsov/Lesson 1/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Резюме</title>
+        <link rel="stylesheet" type="text/css" href="resume.css">
+    </head>
+    <body>
+        <section>        
+            <div class="description">
+                <img class="foto" src="myfoto.jpg" width="230" height="230" alt="">
```
Comment (Polik32) [inline home/Volodymyr Shevtsov/Lesson 1/index.html]: photo

File: home/Volodymyr Shevtsov/Lesson 1/index_en.html (hunk 1 above)
Comment (ShevtsovVolodymyr) [reply home/Volodymyr Shevtsov/Lesson 1/index_en.html]: thx, done

File: home/Volodymyr Shevtsov/Lesson 1/index.html (hunk 2 above)
Comment (ShevtsovVolodymyr) [reply home/Volodymyr Shevtsov/Lesson 1/index.html]: thx, done

**Pull request merged**